docs/src/undefined-behaviour.md

```diff
@@ -44,6 +44,4 @@ Kani makes a best-effort attempt to detect some cases of UB:
     * Kani can detect invalid dereferences, but may not detect them in [place expression context](https://doc.rust-lang.org/reference/expressions.html#place-expressions-and-value-expressions).
 * Invoking undefined behavior via compiler intrinsics.
     * See [current support for compiler intrinsics](./rust-feature-support/intrinsics.md).
-* Producing an invalid value, even in private fields and locals. 
-    * Kani provides a [mechanism](./tutorial-nondeterministic-variables.md#safe-nondeterministic-variables-for-custom-types) `is_valid()` which users can use to check validity of objects, but it does not currently apply to all types.
```

Comment:
This item is repeated because it's a "class of UB that Kani does not detect" in general, but at the same time it "makes a best-effort to detect some cases" of it. So it's being repeated in different contexts and we should keep it as is.

Reply:
Ah, good catch! I overlooked the different contexts.